scripts/deployHyperProverAndInbox.ts

```diff
@@ -77,13 +77,6 @@ async function main() {
     )[0].args.addr
```

Comment:
The mailbox setting logic was removed from the deployment script, but it's not clear how this configuration is managed now. Please document the new setup process.